### DISCUSSION TITLE
ChatGPTWizard and Delphi 10.4 Update 2 Community edition

### DISCUSSION THREAD

**Discussion by sax15** [AliDehbansiahkarbon/ChatGPTWizard#6] [Q&A]
Hello.
I always get a result object HTTP/1.1 429 Too Many Request. I have entered the API key and it is correct. I've tried at different times an days and always the same result. 
Also, if I set it to the docabale window, after saving my desktop, chatGPT does not appear in the selected place. I use 
Delphi 10.4 Update 2., community edition. What could be wrong?

**Comment by AliDehbansiahkarbon** [comment]
Hi, About the "HTTP/1.1 429 Too many requests" it's up to the OpenAI, maybe you have to pay for the subscription or use another account probably to get a better experience.
And about the Dockable form yes you are right it should be open manually for now, I will fix this in the next update if it's possible.

**Reply by sax15** [reply]
THX. I look forward for the fix.
